readme.md

```diff
@@ -130,6 +130,13 @@ You can add the count of all your private contributions to the total commits cou
 ![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&count_private=true)
 ```
 
+### Specifying Year
+
+You can specify a year and fetch only the commits that were made in that year by passing `&year=YYYY` to the parameter.
+
+```md
+![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&year=2020)
+```
 ### Showing icons
 
 To enable icons, you can pass `show_icons=true` in the query param, like so:
```

Comment:
@vzsky Looks good. I will do a full review this weekend. For now, could you maybe also add the `year` parameter in https://github.com/anuraghazra/github-readme-stats/blob/150e13a786bf9f661c3236ca2ec5eb19ca5d69f4/readme.md#stats-card-exclusive-options.